test/ui-test/src/utils/local_system.py

```diff
@@ -1,43 +1,8 @@
-import os
-import signal
 import subprocess
-import time
 
-import configs
 
-
-def get_pid(process_name: str):
-    pid_list = []
-    for line in os.popen("ps ax | grep " + process_name + " | grep -v grep"):
-        pid_list.append(int(line.split()[0]))
-    return pid_list
-
-
-def kill_process(process_name: str, verify: bool = True, timeout_sec: int = configs.squish.PROCESS_TIMEOUT_SEC):
-    pid_list = get_pid(process_name)
-    for pid in pid_list:
-        os.kill(pid, signal.SIGKILL)
-    if verify:
-        wait_for_close(process_name, timeout_sec)
-
-
-def wait_for_started(process_name: str, timeout_sec: int = configs.squish.PROCESS_TIMEOUT_SEC):
-    started_at = time.monotonic()
-    while True:
-        pid_list = get_pid(process_name)
-        if pid_list:
-            return pid_list
-        time.sleep(1)
-        assert time.monotonic() - started_at < timeout_sec, f'Start process error: {process_name}'
-
-
-def wait_for_close(process_name: str, timeout_sec: int = configs.squish.PROCESS_TIMEOUT_SEC):
-    started_at = time.monotonic()
-    while True:
-        if not get_pid(process_name):
-            break
-        time.sleep(1)
-        assert time.monotonic() - started_at < timeout_sec, f'Close process error: {process_name}'
+def kill_process(pid, verify: bool = True, timeout_sec: int = 10):
+    execute(['kill', '-9', pid], check=verify, timeout_sec=timeout_sec)
```

Comment:
What's wrong with using `os.kill()`?

Reply:
It does not close the app sometimes

Reply:
Okay then, but I would then add a comment explaining why we are using the `kill` command and not `os.kill()` to avoid an introduction of a regression in the future when someone finds this and tries to change it.

As to why `os.kill()` doesn't work, that does not make sense to me. Either the signal is sent or not.

Reply:
I tested it for all OS again, and you right os.kill works the same as executing the kill command, so I reverted changes